epistemix_platform/src/epistemix_platform/use_cases/upload_results.py

```
@@ -0,0 +1,160 @@
+                        arcname = file_path.relative_to(results_dir)
+                    else:
+                        # Single RUN* directory case: prefix with directory name
+                        arcname = Path(results_dir.name) / file_path.relative_to(results_dir)
+                    zip_file.write(file_path, arcname=arcname.as_posix())
+                    logger.debug(f"Added to ZIP: {arcname}")
+
+        zip_buffer.seek(0)
+        zip_size = len(zip_buffer.getvalue())
+        logger.info(f"Created ZIP file: {zip_size} bytes ({zip_size / 1024 / 1024:.2f} MB)")
+
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)